src/base-command.ts

```diff
@@ -47,8 +47,14 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
   }
 
   async setupFlow(skipOrgId = false): Promise<SetupParameters> {
-    const snykToken = process.env.SNYK_TOKEN ?? (await input({message: 'Enter your Snyk Token'}))
-    process.env.SNYK_TOKEN = snykToken
+    if (!process.env.SNYK_TOKEN) {
+      process.env.SNYK_TOKEN = await input({message: 'Enter your Snyk Token'})
+
+      this.log(ux.colorize('yellow', `\nTIPS`))
+      this.log(ux.colorize('yellow', `For smoother experience, please set the Snyk Token as env variable.`))
+      this.log(ux.colorize('yellow', `Linux/Mac: export SNYK_TOKEN=${process.env.SNYK_TOKEN}`))
```

Comment:
enh: would be nice to give prompts for the user's OS only

Reply:
granted. not there yet.